README.md

```diff
@@ -4,254 +4,218 @@
-which will have some tokens (4M of tez in this example) after the chain start. Note
-that all bakers should have some tokens, thus, they should be listed in `bootstrap_accounts`.
+Two sample JSON files are provided, depending on the version of the network you plan to run:
+* ./parameters/parameters_babylonnet.json
+* ./parameters/parameters_carthagenet.json
 
-### Using docker
+In these files, `bootstrap_accounts` has information about account public keys
+that have access to tokens (4M of tez in these example files). Note
+that all bakers should have some tokens, thus, we need to add the public key for the baker just created into `bootstrap_accounts`.
 
-At first, you will need container running the dictator node.
+Starting with the appropriate sample file for the network version you plan to run,
+modify it by adding an entry in the bootstrap_accounts section for each of the two public key provided in the previous step.
+
+e.g. paste entries like this into the bootstrap_accounts section:
+```sh
+    [
+      "edpkvRTXYRCxCbWs4GF1shMxCab9nF3iNimPqqb2esiP5WyjAhT1dz",
+      "4000000000000"
+    ],
+    [
+      "edpkum3W1vGfsF19uNNnjdThGvbTBXbBcKyCmEAuV5TPfensRxYyqA",
+      "4000000000000"
+    ],
+```
+The exisiting bootstrap accounts should remain in the file, and will be used later in this example.
 
-Second step is to copy parameters file to the docker filesystem:
+Copy the edited parameters file to the two docker filesystems:
 ```sh
-docker cp parameters/parameters_carthagenet.json <container_name>:/parameters.json
+docker cp my-parameters.json <container_name>:/parameters.json
+docker cp my-parameters.json <container_name_1>:/parameters.json
 ```
+where my-parameters.json is the file you have just edited. <container_name> and <container_name_1> can be retrieved by the command 'docker ps'
 
-And the last step is to run activation script for running docker container:
+The last step is to run the activation script for the running docker containers.
+For this step, choose the container name corresponding to the first container we created (select the one shown to have been started earliest)
 ```sh
 docker exec <container_name> ./scripts/activate-protocol.sh \
+  -A <container_ip> -P 8732
   --base-dir /base-dir --tezos-client /base-dir/tezos-client \
   --parameters /parameters.json
 ```
+where <container_ip> is the IP address displayed when we ran the first baker
 
-## Using private chain
+```sh
+#TODO: This causes the following error, but seems to succeed anyway:
+# Error:
+#  Unrecognized command.
+```
+
+If you want to browse the file system inside your Docker container, you can run the command:
+```sh
+docker exec -it <container_name> bash
+```
+This will run an interactive shell session inside your running node container.
+
+### Using the private chain
+Before continuing, you can verify things are working properly by entering into your browser:
+```sh
+http://localhost:8732/chains/main/blocks/head
```

Comment:
AFAIR, for the docker version you should used `container_ip` instead of `localhost`, e.g. `172.17.0.2:8732`